src/env-inl.h

```diff
@@ -505,6 +505,22 @@ inline ares_channel* Environment::cares_channel_ptr() {
   return &cares_channel_;
 }
 
+inline bool Environment::cares_query_last_ok() {
+  return cares_query_last_ok_;
+}
+
+inline void Environment::cares_query_set_last_ok(bool ok) {
+  cares_query_last_ok_ = ok;
+}
+
+inline bool Environment::cares_is_servers_use_default() {
+  return cares_is_servers_use_default_;
+}
+
+inline void Environment::cares_servers_use_default(bool is_default) {
+  cares_is_servers_use_default_ = is_default;
+}
```

Comment:
Any chance you could name the setters `set_FLAG()` and the getters just `FLAG()` (compare with e.g. `using_domains`)?